src/routes/notifications/notifications.controller.spec.ts

```diff
@@ -68,23 +68,39 @@ describe('Notifications Controller (Unit)', () => {
         .with('safeRegistration', safeRegistrations)
         .build();
       mockNetworkService.get.mockImplementation((url) => {
-        const getChainUrlPattern = `${safeConfigUrl}/api/v1/chains/`;
-        if (url.includes(getChainUrlPattern)) {
+        if (
+          chains.some(
+            (chain) =>
+              url === `${safeConfigUrl}/api/v1/chains/${chain.chainId}`,
+          )
+        ) {
           return Promise.resolve({ data: sample(chains) });
         }
         return Promise.reject(`No matching rule for url: ${url}`);
       });
-      mockNetworkService.post.mockImplementation(() => Promise.resolve());
+      mockNetworkService.post.mockImplementation((url) => {
+        if (
+          chains.some(
+            (chain) =>
+              url ===
+              `${chain.transactionService}/api/v1/notifications/devices/`,
+          )
+        ) {
+          return Promise.resolve();
+        }
+        return Promise.reject(`No matching rule for url: ${url}`);
+      });
 
       await request(app.getHttpServer())
         .post('/register/notifications')
         .send(registerDeviceDto)
-        .expect(200);
+        .expect(200)
+        .then(({ body }) => expect(body).toEqual({}));
```

Comment:
You can call `expect` since it takes a `body` directly.

```suggestion
        .expect({});
```

same for the other tests.

Reply:
Changed in [48a179c](https://github.com/5afe/safe-client-gateway-nest/pull/267/commits/48a179c6ee1ea8bd2c04c00e567f9a2b97b71aa0)